biojava-core/src/main/java/org/biojava/nbio/core/util/StringManipulationHelper.java

```diff
@@ -174,14 +175,36 @@ public static boolean equalsToXml(String expected, String actual) {
 		throw new UnsupportedOperationException("not yet implemented");
 	}
 
+	/**
+	 * Adds padding to left of supplied string
+	 * @param s The String to pad
+	 * @param n an integer >= 1
+	 * @return The left-padded string. 
+	 * @throws IllegalArgumentException if n <= 0
+	 */
 	public static String padLeft(String s, int n) {
+		validatePadding(n);
 	    return String.format("%1$" + n + "s", s);
 	}
 
+	/**
+	 * Adds padding to right of supplied string
+	 * @param s The String to pad
+	 * @param n an integer >= 1
+	 * @return The right-padded string. 
+	 * @throws IllegalArgumentException if n <= 0
+	 */
 	public static String padRight(String s, int n) {
+		validatePadding(n);
 	     return String.format("%1$-" + n + "s", s);
 	}
 
+	private static void validatePadding(int n) {
+		if (n <=0 ) {
```

Comment:
Why can't the padding be == 0?

Reply:
If I set padding to 0, a java.util.MissingFormatWidthException is thrown from within String.format method when padRight is called, and a FormatFlagsConversionMismatchException when padLeft is called.

Reply:
Thank you for clarification, @richarda23. I doublechecked the Javadoc and it is inhibited from their side.
Please add your name in the @author tag.